src/utils/dayViewLayout.js

```diff
@@ -355,10 +358,56 @@ export function getStyledAvailabilities ({
     totalMin,
   };
 
-  availabilities.forEach((availability, availabilityIdx) => {
-    let style = getYStyles(availabilityIdx, helperArgs);
-    styledAvailabilities.push({ availability, style });
-  })
+  const overlapMap = {};
+  let columnIndex = 0;
+
+  for (let availabilityIndex = 0; availabilityIndex < availabilities.length; availabilityIndex++) {
+    const availability = availabilities[availabilityIndex];
+    let grouped = false;
+
+    // Check for overlap with existing groups
+    const overlapKeys = Object.keys(overlapMap);
```

Comment:
What is an "overlap key"? An ID? A column index? A time? "Overlap" doesn't really say anything.